### DISCUSSION TITLE
Volume Control adjustment to Coarse

### DISCUSSION THREAD

**Discussion by brentlr** [linuxmint/discussions#146] [Ideas]
I was requesting if the Volume Control in Linux mint could be made to make finer adjustments, mine jumps 5 every time I move it up or down, unlike Windows that moves 1 point at a time. Hoping this can be integrated in a future release. Thank you for Consideration

**Comment by AtomicRobotMan0101** [comment]
this will provide the functionality you require: https://cinnamon-spices.linuxmint.com/applets/view/306

BTW, agree on granularity.

**Comment by brentlr** [comment]
How does one install this?
TIA!

**Comment by brentlr** [comment]
Sorry I am a new user tof LINUX Mint, I got the zip file downloaded, what do I do next? Thank you

**Reply by rcalixte** [reply]
You can install this better if you do it via Applets in the settings or menu. Search for this applet by name there and you'll be able to install it and also get updates.

**Reply by AtomicRobotMan0101** [reply]
as @rcalixte says...

**This downloads the applet into the "collection"**

--> Right click the bottom toolbar --> select **Applets** --> click the **Download** tab along the top --> find **150** --> click the **downward pointing arrow** on the right of it.


**Next, activate it**

Go back to the Manage Tab (top tab) --> Find 150 (if its not already selected) and then **click "the "star"** (Add) at the bottom.

It doesn't feel too intuitive, but once done once, it makes sense.

....

There are many truly fantastic Applets in there. The lads at Mint have done a great job on this. When I first started, I thought these were mostly junky add-ons, but now as I progress I see their true value. Some true gems in there.

**My suggestions?**

- Cinnamenu
- ScreenShot+Record Desktop
- Sound 150%
- Workspace switcher (there is a better one, but this is excellent)


(edit - updated instructions to work with Mint 21.3 Victoria :)  )

**Comment by brentlr** [comment]
Hi Evan,
Thank you for your help.
I am using Linux Mint 21.2 Victoria \n \l
Followed your instructions but the Applet app found nothing listed using
@150, I also tried pasting in the Get Hub Link and that did not work
either. Maybe it does not work on Mint 21.2?

Any added thoughts on the install procedure greatly appreciated?

Thank you,

**Reply by rcalixte** [reply]
You can just search for "150" and you should be able to find it.

**Comment by freonheat** [comment]
I came to post the same thing.  Linux Mint 21.2 + Cinnamon...

I have always used the mouse wheel to control volume while hovering over the "Sound" icon in the tray.  However, it jumps in increments of 5% which is way too rough.  +5% or -5% on my system is a LOT of volume.   It does the same thing if I click on the icon and use mouse wheel on the horizontal scroller,  +/-5%.

I addition, when I use the volume up/down buttons on my keyboard, that is ALSO +/-5%!

My suggestion is to have the "Sound" control customizable ("Configure" option) so the user can set the granularity.

**Comment by Secret-chest** [comment]
Use the Sound150 applet.

**Reply by freonheat** [reply]
Yep.  Installed it and it is great.
But my suggestion still stands.  It looks like maybe 150 should be the default for Cinnamon.  There is no advantage to having less functionality.
